RELEASE.md

```diff
@@ -0,0 +1,31 @@
+## Major / Minor Release
+
+- create a branch for the release e.g. `7.6.x`
+- provide a [PR](https://github.com/eclipse/che-machine-exec/pull/66) with bumping the [VERSION](https://github.com/eclipse/che-machine-exec/blob/master/VERSION) file to the `7.6.x` branch
```

Comment:
Might want to clarify which version is used in the file, eg., 7.6.0 or 7.7.0-SNAPSHOT ?

Reply:
updated